data/json/construction.json

```
@@ -2309,5 +2309,18 @@
     "pre_note": "Can be deconstructed without tools.",
     "pre_terrain": "t_pit_shallow",
     "post_terrain": "f_firering"
+  },
+ {
+    "type": "construction",
+    "description": "Build Rammed Earth Wall",
+    "//": "No wood required! Cleans up dirt quick! Frame is used only to shape the earth, and can be reused afterward.",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 2 ], [ "survival", 2 ] ],
+    "time": 300,
+    "qualities": [ { "id": "HAMMER", "level": 2 } ],
+    "tools": [ [ "frame_wood_light" ], [ "log" ] ],
+    "components": [ [ [ "material_soil", 100 ] ], [ [ "water", 10 ], [ "water_clean", 10 ] ] ],
+    "pre_terrain": "t_fence_post",
+    "post_terrain": "t_wall_rammed_earth"
   }
-]
+]
```

Need a new line at the end of the file
```suggestion
]

```